config/i18n-tasks.yml

```diff
@@ -111,6 +110,8 @@ ignore_unused:
   - 'user_mailer.email_confirmation_instructions.subject'
   - 'valid_email.validations.email.invalid'
   - 'errors.attributes.*'
+  - 'datetime.dotiw.minutes.*'
+  - 'datetime.dotiw.seconds.*'
```

Comment:
Hm, I wouldn't expect you should need to do this, since that's the point of the `i18n-tasks-use` above. 🤔

Reply:
I dug into this a little bit. The issue seems to be due to how Rails natively understands [pluralization objects](https://guides.rubyonrails.org/i18n.html#pluralization), so in a Rails context, you would translate a string like....

```rb
t('datetime.dotiw.minutes', count: 2)
```

The `i18n-tasks` gem understands this, and so is looking for the usage of the parent key, not the `one`, `other`, etc.

https://github.com/glebm/i18n-tasks/blob/d51327cb48870f984303da55fe1a0605cf0e70d3/lib/i18n/tasks/plural_keys.rb#L6-L20

We don't have the same support for this pluralization object in the JavaScript implementation currently, so I think we'll have to leave it as you have it here for now, and revisit later.